exam-processor/src/main/resources/application.yml

```
@@ -21,4 +21,7 @@ spring:
     testWhileIdle: true
     validationQuery: SELECT 1
     driverClassName: com.mysql.jdbc.Driver
-    initialize: false
\ No newline at end of file
+    initialize: false
+    maxActive: 20
+    initialSize: 4
+    maxWait: 10000
```

the default wait is 30000 (30s) which i've always disliked; if things are so bad that a connection doesn't free up in 10s, please throw an exception